evap/staff/templates/staff_semester_view.html

```diff
@@ -576,18 +584,24 @@ <h3>
     </script>
     <script type="module">
         import {EvaluationGrid, TableGrid} from "{% static 'js/datagrid.js' %}";
-        new EvaluationGrid({
-            storageKey: "evaluation-semester-{{ semester.id }}-data-grid",
-            table: document.querySelector("#evaluation-table"),
-            searchInput: document.querySelector("input[name=search-evaluation]"),
-            filterButtons: [...document.querySelectorAll("#evaluation-filter-buttons [data-filter]")],
-            resetSearch: document.querySelector("[data-reset=search-evaluation]"),
-        }).init();
-        new TableGrid({
-            storageKey: "course-semester-{{ semester.id }}-data-grid",
-            table: document.querySelector("#course-table"),
-            searchInput: document.querySelector("input[name=search-course]"),
-            resetSearch: document.querySelector("[data-reset=search-course]"),
-        }).init();
+        const evaluationTable = document.querySelector("#evaluation-table");
+        if (evaluationTable) {
+            new EvaluationGrid({
+                storageKey: "evaluation-semester-{{ semester.id }}-data-grid",
+                table: document.querySelector("#evaluation-table"),
```

Comment:
```suggestion
                table: evaluationTable,
```

same with `courseTable` below